web/src/features/map/map-layers/StatesLayer.tsx

```diff
@@ -0,0 +1,60 @@
+import { Layer } from 'react-map-gl';
+
+import { MapStyle } from '../mapTypes';
+
+export default function StatesLayer({ mapStyles }: { mapStyles: MapStyle }) {
+  return (
+    <>
+      <Layer
+        id="states-border"
+        type="line"
+        paint={mapStyles.statesBorder}
+        minzoom={2.5}
+      />
+      <Layer
+        id="state-labels-name"
+        type="symbol"
+        source="states"
+        layout={{
+          'text-field': ['get', 'stateName'],
+          'symbol-placement': 'point',
+          'text-size': 12,
+          'text-letter-spacing': 0.12,
+          'text-transform': 'uppercase',
+
+          'text-font': ['poppins-semibold'],
+        }}
+        paint={{
+          'text-color': 'white',
+          'text-halo-color': '#111827',
+          'text-halo-width': 0.5,
+          'text-halo-blur': 0.25,
+          'text-opacity': 0.9,
+        }}
```

Comment:
You could also consider extracting these (layout and styling) to a constant, to avoid this inline-style. That would also allow reusing the text layout configuration something like this:
```
layout={{
  ...textLayout,
  'text-field': ['get', 'stateId'],
}}
```

Reply:
Started doing this one but felt very styled-component-y. I think given this file is quite small inline works better to match our tailwind style of styling

Reply:
Actually I've gone back to your suggestion now I've moved the zone layer out and this styles to the top pattern works for all the layers 😅